src/codeflare_sdk/cluster/auth.py

```diff
@@ -106,10 +106,24 @@ def login(self) -> str:
             configuration.api_key_prefix["authorization"] = "Bearer"
             configuration.host = self.server
             configuration.api_key["authorization"] = self.token
-            if self.skip_tls == False and self.ca_cert_path == None:
+            ca_path_env = os.environ.get("CA_CERT_PATH")
```

Comment:
os.environ.get("CF_SDK_CA_CERT_PATH", ca_path_env)

os.environ.get allows defaulting which can be used here. We should also prefix env vars WDYT of `CF_SDK_`

Reply:
Sounds good to me. I should also update the documentation in this PR too